### ISSUE TITLE
Release dbt-metricflow 0.7.0

### ISSUE THREAD

**Issue by tlento** [dbt-labs/metricflow#1105]
Opening this issue to track dependencies for releasing dbt-metricflow 0.7.0, since our package re-adjustments require added testing. This is future work blocked on the release of dbt-core 1.8 AND 1.8 releases of all supported adapters.
```[tasklist]
### Tasks
- [ ] https://github.com/dbt-labs/metricflow/issues/1091
- [ ] Verify dbt-core and all adapters are on 1.8
- [ ] Deploy test builds and verify installation works across adapter types
- [ ] Run release and publish notes with breaking change notification
```

**Comment by plypaul** [comment]
This version was previously released.
